server/backup_runner.go

```diff
@@ -19,13 +22,17 @@ func NewBackupRunner(server *Server, backupRunnerInterval time.Duration) *Backup
 	return &BackupRunner{
 		server:               server,
 		backupRunnerInterval: backupRunnerInterval,
+		downloadBinlogTasks:  make(map[int]bool),
 	}
 }
 
 // BackupRunner is the backup runner scheduling automatic backups.
 type BackupRunner struct {
 	server               *Server
 	backupRunnerInterval time.Duration
+	downloadBinlogTasks  map[int]bool
```

Comment:
The word "tasks" has never appeared before. downloadBinlogInstanceIDs

Reply:
SG.